test-galasactl-ecosystem.sh

```diff
@@ -332,6 +333,44 @@ function runs_get_check_details_format_output {
     success "galasactl runs get --format details seemed to work"
 }
 
+#--------------------------------------------------------------------------
+function runs_get_check_raw_format_output {
+    h2 "Performing runs get with raw format..."
+
+    run_name=$1
+
+    cd ${BASEDIR}/temp
+
+    cmd="${BASEDIR}/bin/${binary} runs get \
+    --name ${run_name} \
+    --format raw \
+    --bootstrap ${bootstrap} "
+
+    info "Command is: $cmd"
+
+    output_file="runs-get-output.txt"
+    $cmd | tee $output_file
+
+    # Check that the full test name is output
+    cat $output_file | grep "${GALASA_TEST_NAME_LONG}" -q
+    rc=$?
+    # We expect a return code of '0' because the test name should be output.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Did not find ${GALASA_TEST_NAME_LONG} in raw output"
+        exit 1
+    fi  
+
+    # Check that we got 10 pipes
+    pipe_count=$(grep -o "|" $output_file | wc -l | xargs)
```

Comment:
that's not getting a pipe count. It's counting the lines which have pipes in. wc -l counts lines.